Python.md

```diff
@@ -40,7 +85,37 @@ result = connection.execute(sqlalchemy.text("""
            show_gravatar
       FROM "user"
      WHERE musicbrainz_id IN :musicbrainz_usernames
-"""), {
-    'musicbrainz_usernames': tuple(usernames),
-})
+""")
+result = connection.execute(query, {'musicbrainz_usernames': tuple(usernames)})
 ```
+
+It is preferred to assign the query to a separate variable and not include it inside the
+`execute` call.
```

Comment:
OK, let me think on this. I still think it "looks" overly complex, but I realise it's no more than a personal opinion.